webdriver/tests/bidi/script/evaluate/remote_values.py

```diff
@@ -15,3 +15,108 @@ async def test_remote_values(bidi_session, top_context, expression, expected):
     )
 
     recursive_compare(expected, result)
+
```

Comment:
Same comments as for the other file apply here as well. Also there is a linter failure due to the missing empty line at the end of the file.